src/go/k8s/pkg/resources/resource_integration_test.go

```diff
@@ -374,3 +374,53 @@ func TestEnsure_NodePortService(t *testing.T) {
 		t.Fatalf("expecting configmap updated but got %d", port)
 	}
 }
+
+func TestEnsure_LoadbalancerService(t *testing.T) {
+	t.Run("create-loadbalancer-service", func(t *testing.T) {
+		cluster := pandaCluster()
+		cluster = cluster.DeepCopy()
+		cluster.Spec.Configuration.KafkaAPI = append(cluster.Spec.Configuration.KafkaAPI,
+			[]redpandav1alpha1.KafkaAPI{
+				{
+					Port: 1111,
+				},
+				{
+					External: redpandav1alpha1.ExternalConnectivityConfig{
+						Enabled: true,
+						Bootstrap: &redpandav1alpha1.LoadBalancerConfig{
+							Annotations: map[string]string{"key1": "val1"},
+							Port:        2222,
+						}}}}...)
```

Comment:
NIT: Why this indentation is so weird :) I couldn't catch by the first look what is created here. 

I would suggest to not inline closing brackets.

Reply:
agree, adjusted